RelationalAI/Client.cs

```diff
@@ -554,7 +560,7 @@ private string GenLoadJson(string relation)
             var inputs = new Dictionary<string, string>();
             inputs.Add("data", data);
             var source = GenLoadJson(relation);
-            return ExecuteV1(database, engine, source, false, inputs);
+            return ExecuteV1Async(database, engine, source, false, inputs);
```

Comment:
it's a bit strange to see here V1 🤔

Reply:
@NRHelmi is there a specific reason we need V1 Execute here for load CSV & JSON?

Reply:
the sdk still supports v1 execute and one of the usage is loading csv and json data.
we are following a naming convention over all sdks to name the v1 execute/query as `ExecuteV1`

Reply:
That's fine, just wondering wouldn't it be better to use the newer API if it supports the same?

Reply:
v2 is not yet fully implemented API side that's why we still have some similar workarounds in the SDK: for we could use v2 `Execute` to load json and csv data over the wire but this is not how the v2 protocol is designed, I believe the right path is to use multipart forms the same way we are receiving v2 responses but this is not yet supported API side